postman/collection-dir/cybersource/Flow Testcases/Variation Cases/Scenario17- Setup_future_usage is null for normal payments/Payments - Create/request.json

```diff
@@ -0,0 +1,79 @@
+{
+  "method": "POST",
+  "header": [
+    {
+      "key": "Content-Type",
+      "value": "application/json"
+    },
+    {
+      "key": "Accept",
+      "value": "application/json"
+    }
+  ],
+  "body": {
+    "mode": "raw",
+    "options": {
+      "raw": {
+        "language": "json"
+      }
+    },
+    "raw_json_formatted": {
+      "amount": 6540,
+      "currency": "USD",
+      "confirm": false,
+      "capture_method": "automatic",
+      "capture_on": "2022-09-10T10:11:12Z",
+      "amount_to_capture": 6540,
+      "customer_id": "customer{{$randomAlphaNumeric}}",
```

Comment:
Are we using random customerId instead of the default one (`{{customer_id}}`)?

Reply:
yes , to create new customer everytime to check the savecard for new customer